dev/tests/static/testsuite/Magento/Test/Integrity/Layout/BlockNamesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Test block names exists
```

Comment:
Copyright header should be static across all files and such comment should be placed in class PHPDoc.

Feel free to not change this now as there are a lot of such occurrences in code, it needs to be enforced and fixed once for all codebase.

Reply:
Could you fix them during merging this PR?

Reply:
I don't think it's worth neither mine nor your additional commit, just keep in mind for future contributions and I hope to contribute such PHPCS sniff to make all copyright headers and class comments consistent.